Update documentation to include multiple methods for trace methods (#20885)
DataDog/documentation: content/en/tracing/trace_collection/custom_instrumentation/java.md

java -javaagent:/path/to/dd-java-agent.jar -Ddd.env=prod -Ddd.service.name=db-app -Ddd.trace.methods=store.db.SessionManager[saveSession] -jar path/to/application.jar
```

> For tracing several functions within the same class, you should use the following syntax:

Reviewer: ```suggestion
To trace several functions within the same class, use the following syntax:
```
